### PR TITLE
Sentieon T/N and Tumor only pipeline updated 

### PR DESCRIPTION
**Pull request by imsarath**
### This PR:

- TNscope and DNAscope added to sentieon rules
- TNscope filtering based on a machine learning model added
- DNAscope filtering disabled due to machine learning model issues
- Benchmark dir added to capture the processing time and CPU memory usage.
- Fixed fastp rule to write output json and html file for each sample.

### How to test:

- Automatic and continuous test pass

### Expected outcome:
- Installation, unit and integration tests pass

### Review:
- [x] Code review
- [x] New code is executed and covered by tests
- [x] Tests pass

### CODE REVIEW

**Comment by hassanfa** [comment]
Looks good. I'll review it this weekend. A quick comment to add downloading ML models either to reference or to assets.

**Comment by imsarath** [comment]
Now I will add ML models to assets directory, Once we fix the singularity PR, we will move this into the reference workflow flow (Just to avoid conflicts).

File: BALSAMIC/commands/run/analysis.py [hunk 1]
```diff
@@ -123,6 +124,7 @@ def analysis(context, snake_file, sample_config, run_mode, cluster_config,
 
     # Create result directory
     os.makedirs(resultpath, exist_ok=True)
+    os.makedirs(benchmarkpath, exist_ok=True)
```
Comment (hassanfa) [inline BALSAMIC/commands/run/analysis.py]: Move this under `if not...` in next line. It shouldn't create benchmark dir if in dry run mode.

File: BALSAMIC/commands/run/analysis.py [hunk 2]
```diff
@@ -111,6 +111,7 @@ def analysis(context, snake_file, sample_config, run_mode, cluster_config,
     logpath = sample_config['analysis']['log']
     scriptpath = sample_config['analysis']['script']
     resultpath = sample_config['analysis']['result']
+    benchmarkpath = sample_config['analysis']['benchmark']
```
Comment (hassanfa) [inline BALSAMIC/commands/run/analysis.py]: Move this after line 121 `if files:`

**Pull request merged**